PixelHunter1995/TilesetLib/AnimationTileset.cs

```diff
@@ -0,0 +1,142 @@
+﻿using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Graphics;
+using System;
+
+namespace PixelHunter1995.TilesetLib
+{
+    class AnimationTileset : Tileset
+    {
+        // Using hard coded values asuming all animation tilesets look the same
+        private static readonly int IMAGE_WIDTH = 999; // value currently not used...
+        private static readonly int IMAGE_HEIGHT = 999; // value currently not used...
+        private static readonly int FIRST_GID = 0;
+        private static readonly int TILE_WIDTH = 32;
+        private static readonly int TILE_HEIGHT = 48;
+        private static readonly string NAME = "dummy_name"; // value currently not used...
+        private static readonly int TILE_COUNT = 16;
+        private static readonly int NO_OF_COLUMNS = 4;
+
+        // GID referencing first tile
+        private static readonly int STAND_DOWN = 0;
+        private static readonly int STAND_UP = 1;
+        private static readonly int STAND_RIGHT = 2;
+        private static readonly int MOVE_DOWN = 8;
+        private static readonly int MOVE_UP = 12;
```

Comment:
Where do the values come from (I get that it refers to the row for the animation, but why these rows)? At first it seemed like bit-shifting left, but then 12 appears suddenly, instead of 16. My current guess is that STAND directly follow eachothers (starting at 0), while Move animations has 4 rows between each others (starting at 4). No clue what is between the MOVE rows.

Reply:
Values are from how it is currently in felixia.png. Fabian said all animations will be in that order.